pkg/eestream/rs_test.go

```diff
@@ -20,9 +20,9 @@ func TestRS(t *testing.T) {
 	}
 	rs := NewRSScheme(fc, 8*1024)
 	readers := EncodeReader(bytes.NewReader(data), rs)
-	readerMap := make(map[int]io.Reader, len(readers))
+	readerMap := make(map[int]io.ReadCloser, len(readers))
 	for i, reader := range readers {
-		readerMap[i] = reader
+		readerMap[i] = ioutil.NopCloser(reader)
```

Comment:
still need this?

Reply:
Yes, it's needed. `DecodeReaders()` now expects `io.ReadCloser`s

Reply:
right but doesn't TransformReader already return a ReadCloser?

Reply:
oh alright, yeah it's probably fine. we may need to do it eventually